### ISSUE TITLE
Staging Review finding: Screen reader users may miss important instructions

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104048]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing (ARF) + Accredited Rep Mgmt (ARM)
**Product:** ARF (Accredited Rep Portal MVP) + ARM (Appoint a Rep 2.0)
**Feature:** Form 21-22: Digital Submission (ARM) and Accept/Decline (ARF)

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/appoint-rep/representative-organization](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/representative-organization), there's useful helper text that explains that representatives may work with multiple VSOs, and provides context for why you have to choose a VSO. This helper text isn't programmatically associated with the form input, however, so when a screen reader user tabs to the form inputs they may not hear that text. Like anyone else moving through a form flow, many screen reader users will try to move through the flow as efficiently as they can, which means they may not have their screen reader read out the full page content and may instead skip straight to the next input.

<details><summary>Link, screenshot or steps to recreate</summary>

!["Select the organization" form step with helper text explaining the purpose of the question.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/99796-Brian-2.png?raw=true)

</details>

## Recommended action

The easiest solution here is to wrap the whole thing in a `fieldset`, and put all of the relevant helper text inside the `legend`. Note that you can style legend text so visually there won't be any changes.

You'll end up with something like:

```
<fieldset>
   <legend>
      <h3 class="some-class">Select the organization</h3>
      <p class="some-other-class">This accredited VSO...</p>
      <p class="some-other-class">Note: You'll usually work...</p>
   </legend>

   <va-radio ... />
</fieldset>
```

By placing all of the important text in the `legend`, you ensure that it will be announced the first time you tab into the radio buttons.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
